tests/test_taxi.py

```diff
@@ -0,0 +1,88 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+import os.path
+
+from taxi.models import Car, Manufacturer
+
+
+class AminSiteDriverTests(TestCase):
+    def setUp(self) -> None:
+        self.client = Client()
```

Comment:
for not typing this every time you can call `client = test.APIClient()` at once before the starting of your test classes. But I am not sure that it is a best practice